Documentation/architecture.rst

```diff
@@ -259,6 +259,14 @@ Tunnel                   node             64k             Max 32k nodes (IPv4+IP
 IPv4 Fragmentation       node             8k              Max 8k fragmented datagrams in flight simultaneously on the node
 ======================== ================ =============== =====================================================
 
+For some BPF maps, the upper capacity limit can be overridden using command
+line options for ``cilium-agent``. A given capacity can be set using
+``--bpf-ct-global-tcp-max``, ``--bpf-ct-global-any-max``,
+``--bpf-nat-global-max``, ``--bpf-policy-map-max``, and
+``--bpf-fragments-map-max``. Using ``--bpf-map-dynamic-size-ratio`` these
+upper capacity limits are determined based on the given ratio of the total
+system memory.
```

Comment:
This sentence might be a bit misleading. `--bpf-map-dynamic-size-ratio` doesn't compute a limit for the fragment maps mentioned in the previous sentence.

Reply:
Good point. I'll wait whether there are any other review comments and address this in batch.

Reply:
Now fixed. Also took the opportunity to slightly extend the paragraph about `--bpf-map-dynamic-size-ratio`.